source/core_mqtt.c

```diff
@@ -733,35 +733,37 @@ static int32_t sendMessageVector( MQTTContext_t * pContext,
                                   TransportOutVector_t * pIoVec,
                                   size_t ioVecCount )
 {
-    uint32_t timeoutTime;
-    uint32_t bytesToSend = 0U;
-    int32_t bytesSentOrError = 0;
+    int32_t sendResult;
+    uint32_t lastSendTimeMs;
+    uint32_t timeSinceLastSendMs;
     TransportOutVector_t * pIoVectIterator;
     size_t vectorsToBeSent = ioVecCount;
+    size_t bytesToSend = 0;
```

Comment:
There should a `U` at the end here. That is, it should be:
`size_t bytesToSend = 0U;`

Reply:
Added the ```U``` to this.